cmd/envbuilder/main.go

```diff
@@ -23,15 +23,14 @@ import (
 )
 
 func main() {
-	root := &cobra.Command{
-		Use: "envbuilder",
+	config := envbuilder.Config{}
```

Comment:
nit: alternatively you can write it as:

`var config envbuilder.Config`

Reply:
What is the difference and why is this preferred?

Reply:
`var x Thing` is just delcaration https://go.dev/ref/spec#Variable_declarations
`x := Thing` is both declaration and assignment https://go.dev/ref/spec#Short_variable_declarations

IMO it's dealer's choice.

Reply:
I'm fine with leaving as is!